ReleaseNotes.txt

```diff
@@ -1,83 +0,0 @@
-
```

Comment:
i know you want to restart releasenotes but iah theres no point because anyone willing to read the release notes for this will be willing to read the PRs. cool with this